src/app/contextBroker/contextBroker.cpp

```diff
@@ -243,6 +243,7 @@ unsigned        cprForwardLimit;
 int             subCacheInterval;
```

Comment:
CNR entry missing.

In addition, cli.md could include the following entry regarding the new `-connectionMemory`:

"""
-connectionMemory, set the size of the connection memory buffer (in Kb) [is "connection memory buffer" correct? otherwise use the right term] used internally by the HTTP server library. Default value is 64 Kb.
"""